ssl-cert-check

```diff
@@ -51,7 +52,7 @@ if [[ "$1" =~ ^--(list|glob)=.* ]]; then
     if [[ ! -r "$LIST" ]]; then
         echo "Error: list is not a regular file" >&2
     fi
-    readarray -t TARGETS < <(grep -v '^#' "$LIST")
+    readarray -t TARGETS < <(grep -v '\(^#\|^\W*$\)' "$LIST")
```

Comment:
As I understand `\W` in POSIX regex, this would ignore lines full of non-word characters (something like `@#.!/`). Shouldn't this be `^\s*$` to match whitespace only?